django_readers/pairs.py

```diff
@@ -42,6 +43,24 @@ def field_display(name):
     )
 
 
+def count(name, distinct=True):
+    attr_name = f"{name}_count"
+    return (
+        qs.annotate(**{attr_name: Count(name, distinct=distinct)}),
+        projectors.attr(attr_name),
+    )
+
+
+def exists(name, distinct=True):
+    attr_name = f"{name}_count"
+    return (
+        qs.annotate(**{attr_name: Count(name, distinct=distinct)}),
+        projectors.alias(
+            f"{name}_exists", projectors.attr(attr_name, transform_value=bool)
```

Comment:
`name` would usually be plural wouldn't it? `cats_exists` is a slightly odd naming scheme. `f"{name}_exist"` would give `cats_exist` which is more natural perhaps? but then it differs from `exists()` and `Exists()` by only one character which is a recipe for confusion.

Usually when doing this manually I call the key `has_cats` so how about `f"has_{name}"`?